server/routes/service.test.js

```diff
@@ -92,137 +93,146 @@ describe('Serivice layer unit test', () => {
+    sinon.stub(db, 'createUser').returns({ user_id: 9999, user_name: null });
 
-    const result = await service.signUpDbQuery('correct_name', 'correct_pw');
+    const result = await service.signUpDbQuery('nonexisting_name', 'nonexisting_pw');
 
-    expect(result).to.equal('accessToken');
+    expect(result).to.equal('500');
   });
 
-  it('Check new user record created but user id not provided and returns 500', async () => {
-    sinon.stub(db, 'createUser').returns({ user_id: null, user_name: 'correct_name' });
+  it('Should return 500 and fail creating new user for null user id', async () => {
+    sinon.stub(db, 'createUser').returns({ user_id: null, user_name: 'nonexisting_name' });
 
-    const result = await service.signUpDbQuery('correct_name', 'correct_pw');
+    const result = await service.signUpDbQuery('nonexisting_name', 'nonexisting_pw');
 
     expect(result).to.equal('500');
   });
 
   // Test for function tokenCheck
-  it('Check cookie contains token information and returns if it is null', async () => {
+  it('Should return 401 for null cookie', async () => {
     const result = service.tokenCheck(null);
 
     expect(result).to.equal('401');
   });
 
-  it('Check an inputted sign in information is invalid and emmits error and returns 403', async () => {
-    sinon.stub(jwt, 'verify').returns('403');
+  it('Should return 403 for invalid user input', async () => {
+    sinon.stub(jwt, 'verify').withArgs('incorrect_information').returns('403');
 
     const result = service.tokenCheck('incorrect_information');
 
     expect(result).to.equal('403');
   });
 
-  it('Check an inputted sign in information is valid and returns user information', async () => {
-    sinon.stub(jwt, 'verify').returns('user_information');
+  it('Should return user information for valid sign in user input', async () => {
+    sinon.stub(jwt, 'verify').withArgs('correct_information').returns('user_information');
 
     const result = service.tokenCheck('correct_information');
 
     expect(result).to.equal('user_information');
   });
 
-  // Test for function queryList
-  it('Check signed in user list retrieved from the database', async () => {
-    sinon.stub(db, 'getList').returns('lists');
+  // Test for function getListByUser
+  it('Should return item lists for the valid user id', async () => {
+    sinon.stub(db, 'getList').returns(['correct','updated','list']);
 
-    const result = await service.queryList('correct_userId');
+    const result = await service.getListByUser('correct_userId');
 
-    expect(result).to.equal('lists');
+    expect(result).to.be.an('array');
+    expect(result).to.have.lengthOf(3);
+    expect(result).to.deep.equal(['correct', 'updated', 'list']);
   });
 
-  it('Check signed in user list retrieved null and returns 500', async () => {
-    sinon.stub(db, 'getList').returns(null);
+  it('Should return 500 for no list retrieved from the database', async () => {
```

Comment:
`should return 500 if no lists are retrieved from the database`

But also, is it really returning a 500?  It looks like it will just be an empty array.  The test name should probably be updated to reflect that